deploy/kubernetes/releases/csi-digitalocean-v4.14.0/driver.yaml

```diff
@@ -0,0 +1,496 @@
+# Copyright 2023 DigitalOcean
+#
```

Comment:
The updates you made for the 4.14 version should also go in the dev manifests (deploy/kubernetes/releases/csi-digitalocean-dev)